### ISSUE TITLE
Attempt to limit Invariant Violations

### ISSUE THREAD

**Issue by smashah** [open-wa/wa-automate-nodejs#3121]

**Comment by smashah** [comment]
@github-actions run

<details>
<summary>⚡ Release!  ⚡</summary>

```js
(async () => {
function exec(cmd) {
try {
	execSync(cmd, {stdio: 'inherit'})
}  catch (error) {
    console.log(`Status Code: ${error.status} with '${error.message}'`);
 }
}
//set the version type
process.env.VERS = "patch"

// Config
const gitUserEmail = "github-actions[bot]@users.noreply.github.com";
const gitUserName = "github-actions[bot]";

exec(`echo "//registry.npmjs.org/:_authToken=$NPM_TOKEN" > .npmrc`);
exec(`git config --global user.email "${gitUserEmail}"`);
exec(`git config --global user.name "${gitUserName}"`);
exec(`npm i -D`);
exec(`npm run release-ci $VERS`);

// types only package
exec('npm config set //registry.npmjs.org/:_authToken $NPM_TOKEN')
exec(`npm config set //registry.npmjs.org/:_authToken $NPM_TOKEN && npm run types:publish `);
exec(`git commit -a -m 'updated types-only package'`);
exec(`git push --force`);

//comment on the issue
var result = execSync(`npx auto-changelog -o ./tempchangelog.txt --commit-limit false --template ./compact-keepachangelog.hbs --stdout`).toString();

    await postComment(result);

//create changelog image
exec(`npm run release-image`);
exec(`git commit -a -m 'updated release-image'`);
exec(`git push --force`);
  })();
```
</details>

**Comment by smashah** [comment]
# Changelog

## 🚀 Release [4.66.0](https://github.com/open-wa/wa-automate-nodejs/compare/4.65.1...4.66.0) (2023-05-11)

- Release 4.66.0 [`9c28b84`](https://github.com/open-wa/wa-automate-nodejs/commit/9c28b8416e36c9eeba86ad65a5108b4e2963a592)
- ⬆️  upgrade uuid dep #3115 [`f9ef8bd`](https://github.com/open-wa/wa-automate-nodejs/commit/f9ef8bd6239b979c26a35922e22065ce1f08b467)
- ⬆️ upgrade discord collection dep #3115 [`ec6561a`](https://github.com/open-wa/wa-automate-nodejs/commit/ec6561a60d748a84e4ffb88a4cb7c2458569e3d6)
- 🗑️ depreacted lists and buttons till further notice #3116 [`45594e0`](https://github.com/open-wa/wa-automate-nodejs/commit/45594e0f1a40e2a70335a6ac659e80c4bdc80681)
- updated types-only package [`f7d172d`](https://github.com/open-wa/wa-automate-nodejs/commit/f7d172deade94cae3551cf461ab3b627e37f9479)
- 💚 fix build [`fa54448`](https://github.com/open-wa/wa-automate-nodejs/commit/fa54448e608f509cc5b5e7edd9a75aab02a9c011)
- 🥚 patches update [`ef6779b`](https://github.com/open-wa/wa-automate-nodejs/commit/ef6779b032b2dbb2f3075f3a68631eb209881ca5)
- updated types-only package [`d79d35d`](https://github.com/open-wa/wa-automate-nodejs/commit/d79d35d55936a535ebe9f94533eef073f8bf8ba7)

**Comment by smashah** [comment]
# Changelog

## 🚀 Release [4.66.3](https://github.com/open-wa/wa-automate-nodejs/compare/4.66.0...4.66.3) (2023-05-20)

- 💚 fix discord dep issues (revert) #3115 [`ef91e41`](https://github.com/open-wa/wa-automate-nodejs/commit/ef91e4189814fb792564fc639315b1d7b3446872)
- Release 4.66.3 [`1415f74`](https://github.com/open-wa/wa-automate-nodejs/commit/1415f749b51eae0383d78ea43b16e82db0a048f1)
- 🚸 Invariant violation avoidance measures #3121 [`ef2dcff`](https://github.com/open-wa/wa-automate-nodejs/commit/ef2dcffb7769a5ab2e23144e22f2d5f03e939543)
- updated types-only package [`561b5f3`](https://github.com/open-wa/wa-automate-nodejs/commit/561b5f32ec9d6148df9941216e108d427f2b9ade)
- updated types-only package [`172284a`](https://github.com/open-wa/wa-automate-nodejs/commit/172284af8b0185062eb7b3f8aa08cdb038c57ecd)
- 🚑 fix collector import [`1ff6282`](https://github.com/open-wa/wa-automate-nodejs/commit/1ff6282b5224aacab9776c7671e56b8b9ea96f05)
- 🥚 patches update [`318b934`](https://github.com/open-wa/wa-automate-nodejs/commit/318b934b0cda7ec360664a9c2c4b59a0a95b8907)
- updated types-only package [`3c57208`](https://github.com/open-wa/wa-automate-nodejs/commit/3c572084d5ac694601f6c9614ed79d99253c685b)
